pkg_api/utils.py

```diff
@@ -107,48 +108,7 @@ def _get_concept_representation(concept: Concept) -> str:
     return _clean_sparql_representation(representation)
 
 
-def _get_preference_representation(
```

Comment:
How come we don't need it anymore?

Reply:
The insertion of a preference is not separated from the insertion of a statement. Preference are now only used in `get_query_for_conditioned_get_preference` and `get_query_for_add_preference` and you can observe that their representation does not include the same properties, so I decided to remove this function.

Reply:
OK, got it. Thank you!